src/utils/spo.ts

```diff
@@ -1769,5 +1769,44 @@ export const spo = {
 
     const itemsResponse = await request.get<ListItemInstance>(requestOptionsItems);
     return (itemsResponse);
+  },
+
+  /**
+   * Gets the site collection URL for a given web URL using SP Admin site.
+   * @param adminUrl The SharePoint admin URL
+   * @param siteId The site ID
+   * @returns Owner login name
+   */
+  async getPrimaryAdminLoginNameAsAdmin(adminUrl: string, siteId: string): Promise<string> {
```

Comment:
Normally, when we create a util or add a function to a util, we add the tests for this function in the util spec file. Could you do this for these functions too? And then in the command spec file you can simply use `spo.getPrimaryAdminLoginNameAsAdmin().resolves('')`.

Reply:
Done